src/Console/Application.php

```diff
@@ -170,11 +165,6 @@ protected function doRunCommand(Command $command, InputInterface $input, OutputI
     {
         $this->command = $command;
 
-        if ('_complete' !== $command->getName() && !class_exists(\RepackedApplication::class)) {
```

Comment:
Options are not bound here, so we must move to a listener. More over it's cleaner, I like